sites/bizbash/server/templates/content/index.marko

```diff
@@ -37,6 +37,11 @@ $ const displayPrimaryImage = ['whitepaper', 'media-gallery'].includes(content.t
     <div class="row">
       <div class="col-lg-8">
         <endeavor-content-block-page-body content=content display-primary-image=displayPrimaryImage />
+        <if(content.type === 'article')>
+          <bizbash-featured-venues-suppliers
```

Comment:
Combine L41 and L42 into single line.

Also - just want to confirm that this should appear on _all_ articles?